clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -199,11 +199,12 @@ export class SettingsService {
     ];
     paths
       .filter((e) => !path.isAbsolute(e))
-      .forEach((localPath) => {
+      .forEach((localPath, index) => {
         const wsFolders = SettingsUtils.getWorkspaceFoldersPath(true);
         wsFolders.forEach((wsFolder) => {
           paths.push(path.join(wsFolder, localPath));
         });
+        paths.splice(index, 1);
```

Comment:
Please extract this transformation as a `function(path, wsFolders)` and add tests for it, including the following case:
```
[
'/absolute',
'relative'
]
```

Reply:
should do the work : d7008215dfc966ad950e1b95978d757019937191